eduvarjaor/decentral-home
# Create a user interface for dapp

## Description

Create a user interface for dapp.

## **Related Issue:**

N/A

## **Main changes explained:**

- Initiate a Next setup 
- Create file App.tsx, style and configure
- Create file Header.tsx, style and configure
- Create file Properties.tsx, style and configure

## **How to test:**

1. check into current branch
2. cd client
3. do `npm install` to run this PR locally
4. run `npm run dev`

## **Screenshots or videos of changes:**

**Result:**

If the user interface opens correctly, everything worked as intended. 

**Screenshots:**

![Screenshot_1](https://github.com/eduvarjaor/decentral-home/assets/86692035/47a3ae24-c578-470d-9c94-3c1616914017)
